cerbero/build/cookbook.py

```diff
@@ -196,14 +196,27 @@ def update_installed_files(self, recipe_name, files):
         @param files: installed files
         @type files: list
         '''
+        # os.path.exists returns False for broken symbolic links
+        def _file_exists(install_dir, file):
+            f = os.path.join(install_dir, file)
+            return os.path.exists(f)
```

Comment:
why don't you use os.path.isfile() here ? because if there is a directory os.remove() below will fail anyway.